client/starwhale/utils/__init__.py

```diff
@@ -148,7 +149,10 @@ def get_downloadable_sw_version() -> str:
 
 def get_field(item: t.Dict, field: str):  # type:ignore
```

Comment:
Is it mean that the type of item is Any?

Reply:
Fixed